ha/cli/exec/clusterExecutor.py

```diff
@@ -66,22 +66,60 @@ def execute(self) -> None:
         Execute the cluster start for all or only cluster
         """
         Log.info("Executing cortxha cluster start")
-        _cluster_start_result = self._cluster_manager.cluster_controller.start()
+        _cluster_start_result = self.cluster_manager.cluster_controller.start()
         if self._args.all:
             Log.info("Executing storage start")
             # TODO : start storage enclosure
 
         if self._args.json:
-            self._op.print_json(_cluster_start_result)
+            self.op.print_json(_cluster_start_result)
         Log.info(_cluster_start_result)
 
 
 class ClusterStopExecutor(CommandExecutor):
+
+    def __init__(self):
+        '''Init method'''
+        super(ClusterStopExecutor, self).__init__()
+        self._args = None
+
+    def parse_args(self) -> None:
+        '''
+           Parses the command line args.
+           Return: argparse
+        '''
+        parser = argparse.ArgumentParser(description="cluster stop")
+        parser.add_argument("cluster", help="Module")
+        parser.add_argument("stop", help="action to be performed")
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument('--all', action='store_true', \
+                            help='Server and storage stop')
+        group.add_argument('--server', action='store_false', \
+                            help='server stop')
+        parser.add_argument('--json', help='Required output format', action='store_true')
```

Comment:
store_true and store_false what is diff ?

Reply:
yeah, we have different options in start I think. Lets understand these.

Reply:
store_false will bydefault results to True and store_true will be True if we give that argument. 
Ex: cortx cluster stop
all: False, server: True
cortx cluster stop --all
all: True, server: True
So, default server will be True which is required.

Reply:
ok

Reply:
Looks good